tests/plugin_tests/test_xkcd.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+import json
+from pathlib import Path
+
+import pytest
+from responses import RequestsMock
+
+DATA_PATH = Path().resolve() / 'tests/data/xkcd'
```

Comment:
If you're going to use pathlib, dont use `/` in `tests/data/xkcx` as it'll break on platforms that dont use `/` as their separator

Reply:
Fixed in d5ff93630bcac309d1832f528d931a721b27b0c3